easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock='Binary'
+
+name = "T-Coffee"
```

Comment:
@verdurin I personally try to consistently use single quotes for words (no spaces) and double qoutes for 'sentences' (strings incl. spaces), but I don't think it's worth blocking a PR over.

There are other issues here though, if this is merged as is, the `develop` is going to fail on the automatic style tests that were enabled recently (cfr. #2506).

See the output of `eb --from-pr 4709 --check-style`, or just close/open this PR to retrigger Travis.